src/config.ts

```diff
@@ -228,23 +358,27 @@ export const Config = Schema.intersect([
 
   PromptConfig,
 
-  Schema.object({
-    output: Schema.union([
-      Schema.const('minimal').description('只发送图片'),
-      Schema.const('default').description('发送图片和关键信息'),
-      Schema.const('verbose').description('发送全部信息'),
-    ]).description('输出方式。').default('default'),
-    allowAnlas: Schema.union([
-      Schema.const(true).description('允许'),
-      Schema.const(false).description('禁止'),
-      Schema.natural().description('权限等级').default(1),
-    ]).default(true).description('是否启用高级功能 (例如图片增强和手动设置某些参数)。'),
-    maxIterations: Schema.natural().description('允许的最大绘制次数。').default(1),
-    maxRetryCount: Schema.natural().description('连接失败时最大的重试次数。').default(3),
-    requestTimeout: Schema.number().role('time').description('当请求超过这个时间时会中止并提示超时。').default(Time.minute),
-    recallTimeout: Schema.number().role('time').description('图片发送后自动撤回的时间 (设置为 0 以禁用此功能)。').default(0),
-    maxConcurrency: Schema.number().description('单个频道下的最大并发数量 (设置为 0 以禁用此功能)。').default(0),
-  }).description('高级设置'),
+  Schema.intersect([
+    Schema.object({
+      output: Schema.union([
+        Schema.const('minimal').description('只发送图片'),
+        Schema.const('default').description('发送图片和关键信息'),
+        Schema.const('verbose').description('发送全部信息'),
+      ]).description('输出方式。').default('default'),
+      maxIterations: Schema.natural().description('允许的最大绘制次数。').default(1),
+      maxRetryCount: Schema.natural().description('连接失败时最大的重试次数。').default(3),
+      requestTimeout: Schema.number().role('time').description('当请求超过这个时间时会中止并提示超时。').default(Time.minute),
+      recallTimeout: Schema.number().role('time').description('图片发送后自动撤回的时间 (设置为 0 以禁用此功能)。').default(0),
+      maxConcurrency: Schema.number().description('单个频道下的最大并发数量 (设置为 0 以禁用此功能)。').default(0),
+    }),
+    Schema.union([
+      Schema.object({
+        type: Schema.const('stable-horde'),
+        trustedWorkerOnly: Schema.boolean().description('是否只请求可信任工作节点。').default(false),
+        pollInterval: Schema.number().role('time').description('轮询进度间隔时长。').default(Time.second),
```

Comment:
I tend to move these two configurations to "login settings", since they seem to be stable horde specific.

Reply:
Sound good. I would change it later.